README.md

```diff
@@ -332,6 +332,10 @@ $ make bench
 
 Issues and pull requests welcome!
 
+## contributors
+
+See package.json
```

Comment:
please turn this into a nice display [like how they do on nan](https://github.com/nodejs/nan/blob/master/README.md#wg-members--collaborators). IMHO this is one of the best node modules and the work done here is important. Contributions to node-nanomsg merit an honorable mention on the readme. :boat: 

I would love to see more contributors added here in the future. And there's still plenty of work to be done here. I can think of two things off the top of my head right now:
* PR for exposing the `ws` socket type!
* Start a `history.markdown` file to track details behind each release. I like how @ronkorving does the history and release management over at zmq: https://github.com/JustinTulloss/zeromq.node/blob/master/History.md We need to start doing that here too. 

woah, for setting up the contributor table, look they're old school using table tags: https://raw.githubusercontent.com/nodejs/nan/master/README.md

i can't stand xml and I try to avoid html table tabs. let's do it [the github markdown way](https://help.github.com/articles/github-flavored-markdown/#tables)